### PR TITLE
Copy changes to Switching Journey

### PR DESCRIPTION
**Pull request by charleycampbell**
## What does this change?
This PR has some copy changes to the switching journey:

- Small print wording on Switch Options for both above and below threshold on any journey
- Switch Options heading wording amend for above and below threshold when on the app
- Switch Review wording amended for both above and below threshold on any journey

## How to test
Check screens in CODE or locally via Storybook (where the below screenshots have been taken) and check that the wording is correct and displaying as expected.

## Images
**App**
Switch Option Above Threshold on app
image

Switch Option Below Threshold on app
image

Switch Review above threshold on app
image

Switch Review below threshold on app
image

-----
**Non-App**
Switch Option Above Threshold non app
image

Switch Option Below Threshold non app
image

Switch Review above threshold non app
image

Switch Review below threshold non app
image

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

**Comment by charleycampbell** [comment]
[MMA: Copy Changes to Switching](https://trello.com/c/N0XW7pJa/3940-mma-copy-changes-to-switching)

File: client/components/mma/switch/options/SwitchOptions.tsx [hunk 1]
```diff
@@ -196,17 +201,16 @@ export const SwitchOptions = () => {
...
 								margin: 0;
 							`}
 						>
-							In exchange for your current payment, you can choose
-							to receive exclusive supporter extras. It takes less
-							than a minute to change your support type and gain
-							access.
+							Your current payment entitles you to exclusive
+							supporter extras. It takes less than a minute to
+							change you support type and gain access.
```
Comment (andrewHEguardian) [inline client/components/mma/switch/options/SwitchOptions.tsx:213]:
```suggestion
							change your support type and gain access.
```

File: client/components/mma/switch/review/SwitchReview.tsx [hunk 2]
```diff
@@ -268,13 +268,13 @@ export const SwitchReview = () => {
...
-							? ". You'll still pay "
-							: ' by paying '}
-						{mainPlan.currency}
-						{formatAmount(newAmount)} per {mainPlan.billingPeriod}.
+							? `Please confirm your choice to get exclusive supporter extras. You'll still pay ${
+									mainPlan.currency
+							  }${formatAmount(newAmount)} per ${
+									mainPlan.billingPeriod
+							  }.`
+							: 'Please confirm your choice to change your support to monthly + extras.'}
```
Comment (andrewHEguardian) [inline client/components/mma/switch/review/SwitchReview.tsx:277]:
```suggestion
							:`Please confirm your choice to change your support to ${monthlyOrAnnual} + extras.`}
```

**Review by jamesmockett — APPROVED** [review]
Looks good 👍

**Pull request merged**